js/modules/k6/http/request.go

```diff
@@ -437,21 +437,25 @@ func (c *Client) prepareBatchObject(requests map[string]interface{}) (
 
 // Batch makes multiple simultaneous HTTP requests. The provideds reqsV should be an array of request
 // objects. Batch returns an array of responses and/or error
-func (c *Client) Batch(reqsV goja.Value) (interface{}, error) {
+func (c *Client) Batch(reqsValues ...goja.Value) (interface{}, error) {
 	state := c.moduleInstance.vu.State()
 	if state == nil {
 		return nil, ErrBatchForbiddenInInitContext
 	}
 
+	if len(reqsValues) == 0 {
+		return nil, fmt.Errorf("no argument was provided to http.batch()")
+	} else if len(reqsValues) > 1 { // Validation for http.batch(["https://validurl"],["https://validurl"])
+		return nil, fmt.Errorf("http.batch() must have only one single array as an argument")
```

Comment:
```suggestion
		return nil, fmt.Errorf("http.batch() must have only one single array as a root argument. Define the requests as nested items.")
```
@olegbespalov WDYT? Do you have better suggestions? I would be very clear here because I think it's easy to waste time making the error to forget the root array.

Reply:
```suggestion
		return nil, fmt.Errorf("http.batch() accepts only an object/array of requests.")
```
Maybe something like this? 

@codebien @vanshaj WDYT?

Reply:
> 

Done updated to "http.batch() accepts only an array or an object of requests." in file and test case also .